Source/Meadow.Foundation.Peripherals/Sensors.Gnss.NeoM8/Driver/Commands.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Meadow.Foundation.Sensors.Gnss
+{
+    static class Commands
+    {
+        /// <summary>
+        /// All data
+        /// </summary>
+        public const string PMTK_SET_NMEA_OUTPUT_ALLDATA =
+            "$PMTK314,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0*28";
+
+        public const string PGCMD_ANTENNA = "$PGCMD,33,1*6C";
```

Comment:
wont these give us XML warnings?

Reply:
yeah they should have .... checking

Reply:
ahh - it's because the class isn't public so they're not exposed to the consumer